app/controllers/topological_inventory/ingress_api/v0x0x2.rb

```diff
@@ -1,7 +1,11 @@
 module TopologicalInventory
   module IngressApi
     module V0x0x2
-      class SchemasController < IngressApi::V0::SchemasController; end
+      class RootController < ApplicationController
+        def openapi
+          render :json => TopologicalInventory::IngressApi::Docs["0.0"].content.to_json
```

Comment:
@Ladas we don't need the `.content` anymore, can you remove in a follow up?

Reply:
https://github.com/ManageIQ/topological_inventory-ingress_api/pull/48/files